module/data/item/spell.mjs

```diff
@@ -186,6 +186,16 @@ export default class SpellData extends ItemDataModel.mixin(ActivitiesTemplate, I
       return obj;
     }, { all: [], vsm: [], tags: [] });
     labels.components.vsm = game.i18n.getListFormatter({ style: "narrow" }).format(labels.components.vsm);
+
+    dnd5e.registry.whenReady(() => {
```

Comment:
It might become a bit hard to reason about Item state if we start scheduling callbacks inside Item preparation. Since we only need this information in the sheet, we should just do this during sheet rendering.